### ISSUE TITLE
improve discovery of actual changes we are inheriting in a release

### ISSUE THREAD

**Issue by ktownsend-personal** [hassio-addons/addon-zwave-js-ui#568]
# Problem/Motivation

This addon is great, but it is very opaque about what underlying zwave-js-ui changes we are getting when this addon updates to a new version of zwave-js-ui. Yes, it is possible to manually observe the version number we are getting and go find info at zwave-js-ui, but general users are unlikely to know where to go or that they even can. This situation could be improved by adding a link to the release notes of zwave-js-ui that shows the underlying changes for the version we are getting. My intent with this request is simply to make the real changes more discoverable for general users.

## Expected behavior

There should be a link in the What's Changed section of the Home Assistant update dialog to the zwave-js-ui change log for the version the addon is updating to. Notable highlights could also be summarized in the addon's change notes.

## Actual behavior

A user wanting to update the dialog has to manually go find the real changes they are going to get, and most users probably don't know where to go. 

## Steps to reproduce

View the Home Assistant update dialog when this addon has an update that includes a newer version of zwave-js-ui.

## Proposed changes

Provide a convenient link to the changes in the new version of zwave-js-ui that will open a new browser tab. Optionally also summarize noteworthy changes.

**Comment by frenck** [comment]
Thanks for the suggestion, however; add-ons ship many more software, not just Z-Wave JS UI.

I am not planning on duplicating the upstream changelogs of all software used in this add-on.

If you want more details, feel free to look upstream (and don’t forget to look upstream from Z-Wave JS UI itself as well, as they don’t duplicate either).

../Frenck

**Comment by ktownsend-personal** [comment]
Mainly recommending a convenient link to the upstream change log to make it more discoverable for general users.

--Keith


On Sep 4, 2023, at 4:59 AM, Franck Nijhof ***@***.***> wrote:

﻿

Thanks for the suggestion, however; add-ons ship many more software, not just Z-Wave JS UI.

I am not planning on duplicating the upstream changelogs of all software used in this add-on.

If you want more details, feel free to look upstream (and don’t forget to look upstream from Z-Wave JS UI itself as well, as they don’t duplicate either).

../Frenck

**Comment by frenck** [comment]
Those links exists, including upstream descriptions in each of the merged pull requests that are in the release (and linked from each release).

**Comment by ktownsend-personal** [comment]
I'll be honest... I know what I'm doing and I still failed to notice the folded line on PR #566<https://github.com/hassio-addons/addon-zwave-js-ui/pull/566> where those links are when I clicked to it from the addon changelog dialog. I only found it just now because you said the links existed and I hunted for it...and I still almost didn't see that subtle arrow indicating folded content. Nice to know that is there for my own convenience, but I wonder if that's too subtle for general users and discoverability could be improved somehow.
